CHANGELOG.md

```diff
@@ -25,9 +25,15 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
 
 ## API changes
 
-### The `type_list` header has moved
+### Argument parser
```

Comment:
Do we want to group things by module? I don't mind, but we should then do it for features, bugs and API changes, right?

Reply:
This gets very nested then altogether. Also, some changes are not module specific, like renaming of Concepts that is going to happen :thinking:

Maybe we can just mention the module in the name of the entry and sort by module?

Reply:
In general the information should be grouped. The most natural grouping would be by module. Then people can review by module whether or not the changes affect their code.